GDJS/Runtime/pixi-renderers/layer-pixi-renderer.js

```diff
@@ -336,9 +339,10 @@ gdjs.LayerPixiRenderer.prototype.getRenderTexture = function () {
  */
 gdjs.LayerPixiRenderer.prototype._replaceContainerWithSprite = function () {
   if (!this._pixiRenderer) return;
-  // @ts-ignore PIXI isn't typed for now.
-  this._lightingSprite = new PIXI.Sprite(this.getRenderTexture());
```

Comment:
`getRenderTexture()` could return a null value, typescript helped in identifying this 😬. So, I changed it a bit to perform a null check (and I'm not calling `getRenderTexture()` anymore because I don't want to assign anything to a new variable, ie, `var rt = this.getRenderTexture()`).